cypress/e2e/tests/migration/dynamic-reports/filter.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    getRandomApplicationData,
+    getRandomAnalysisData,
+    deleteByList,
+    clearAllFilters,
+} from "../../../../utils/utils";
+import { Analysis } from "../../../models/migration/applicationinventory/analysis";
+import { SEC, filterIssue } from "../../../types/constants";
+import { Issues } from "../../../models/migration/dynamicreports/issues";
+let applicationsList: Array<Analysis> = [];
+
+describe(["@tier2"], "Issues filtering", () => {
+    before("Login", function () {
+        login();
+    });
+
+    beforeEach("Load data", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+    });
+
+    it("Running analysis and filtering issues by app name", function () {
+        // For source code analysis application must have source code URL git or svn
+        const application = new Analysis(
+            getRandomApplicationData("bookserverApp", {
+                sourceData: this.appData["bookserver-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["source_analysis_on_bookserverapp"])
+        );
+        application.create();
+        applicationsList.push(application);
```

Comment:
There are lot of filter on applications like app name , BS , tags. I suggest creating 3-4 applications with BS and tags in before method and doing analysis .


Then in test you just use the created data for filtering by app name , BS , tags . That way we won't have to create apps for each test separately.

Reply:
That is my plan to work, but right now, for this PR, I wrote filtering by name only. I will do this in next PRs